Asterix/utils/save_and_read.py

```diff
@@ -249,3 +250,42 @@ def from_param_to_header(config):
         for scalar in config[str(sect)].scalars:
             header[str(scalar)[:8]] = str(config[str(sect)][str(scalar)])
     return header
+
+
+def get_data_dir(env_var_name="THD2_DATA_PATH", config_in=None, datadir="thd2_data"):
+    """
+    Create a path to the local data directory.
+
+    If the environment variable `THD2_DATA_PATH` exists, this is returned as the full data path and the input 'datadir'
```

Comment:
Why THD2_DATA_PATH ? This is not specific to THD2, maybe put ASTERIX_DATA_PATH

Reply:
Ha very correct! I will change this.